[2017/12/20 16:48:10] :: [pool-1-thread-2] :: Base path for the test = C:\New Selenium\Salesforce_QA_Automation\ADT_Resources\<br><br>[2017/12/20 16:48:10] :: [pool-1-thread-2] :: Current Thread Name for the test is "pool-1-thread-2"<br><br>[2017/12/20 16:48:20] :: [pool-1-thread-2] :: The user has launched 'https://qainc--qa.cs30.my.salesforce.com<br><br>[2017/12/20 16:48:21] :: [pool-1-thread-2] :: The user has creating SalesCancelNONorder<br><br>[2017/12/20 16:48:35] :: [pool-1-thread-2] :: arindom.das@move.com.qa has successfully loggedin<br><br>[2017/12/20 16:48:37] :: [pool-1-thread-2] :: Successfully navigated to Home page<br><br>[2017/12/20 16:48:37] :: [pool-1-thread-2] :: File path is = C:\New Selenium\Salesforce_QA_Automation\ADT_Resources\ .<br><br>[2017/12/20 16:50:36] :: [pool-1-thread-2] :: Successfully Account ID is visible= 001j000000bWauw .<br><br>[2017/12/20 16:51:01] :: [pool-1-thread-2] :: Successfully clicked on GlobalSearchBtn .<br><br>[2017/12/20 16:51:36] :: [pool-1-thread-2] :: Successfully clicked on PeopleName .<br><br>[2017/12/20 16:51:55] :: [pool-1-thread-2] :: Successfully clicked on FollowDown .<br><br>[2017/12/20 16:52:08] :: [pool-1-thread-2] :: Successfully clicked on UserDetail .<br><br>[2017/12/20 16:52:27] :: [pool-1-thread-2] :: Successfully clicked on LoginUsers .<br><br>[2017/12/20 16:53:13] :: [pool-1-thread-2] :: Tab is not available,AccountDetails<br><br>[2017/12/20 16:53:17] :: [pool-1-thread-2] :: Successfully switched in to frame<br><br>[2017/12/20 16:53:21] :: [pool-1-thread-2] :: Account id contains primary contact<br><br>[2017/12/20 16:53:54] :: [pool-1-thread-2] :: Successfully clicked on CreateQuote .<br><br>[2017/12/20 16:54:19] :: [pool-1-thread-2] :: Successfully clicked on SaveButton .<br><br>[2017/12/20 16:54:23] :: [pool-1-thread-2] :: Successfully Quote Number is generated- 2017-703175.<br><br>[2017/12/20 16:54:36] :: [pool-1-thread-2] :: Successfully clicked on AddLineItem .<br><br>[2017/12/20 16:54:45] :: [pool-1-thread-2] :: Successfully clicked on Realtor .<br><br>[2017/12/20 16:54:54] :: [pool-1-thread-2] :: Successfully clicked on NonInventoryLink .<br><br>[2017/12/20 16:55:06] :: [pool-1-thread-2] :: Successfully clicked on NonInventoryLinkMap .<br><br>[2017/12/20 16:55:09] :: [pool-1-thread-2] :: Successfully navigated to create Quote<br><br>[2017/12/20 16:55:49] :: [pool-1-thread-2] :: Successfully clicked on LineActionCancel .<br><br>[2017/12/20 16:55:53] :: [pool-1-thread-2] :: Closing the Browser Instance<br><br>[2017/12/20 16:55:53] :: [pool-1-thread-2] ::  File Name: Assert.java Method name: fail(86)<br> File Name: Assert.java Method name: assertTrue(41)<br> File Name: Assert.java Method name: assertTrue(52)<br> File Name: NonInventory.java Method name: clickOnNonInventoryCheckBox(1040)<br> File Name: CancelOrder.java Method name: cancellingNonInventoryOrder(833)<br> File Name: C:/New Selenium/Salesforce_QA_Automation/ADT_Resources/SalesForce/FeatureFiles/AMD1/New_Sales_Cancel_DomainName_NonInventory_With_Credit_Amount2.feature Method name: And I am cancelling non Inventory order and selecting "Domain Name" and "Cancel" and "10" and "Financial Issues" and "Unable to Afford"(11)<br><br>Complete Error Message: null<br><br><br>